bin/src/config.rs

```diff
@@ -7,13 +7,15 @@ struct ConfigFile {
     qmdl_store_path: Option<String>,
     port: Option<u16>,
     readonly_mode: Option<bool>,
+    ui_level: Option<u8>,
 }
 
 #[derive(Debug)]
 pub struct Config {
     pub qmdl_store_path: String,
     pub port: u16,
     pub readonly_mode: bool,
+    pub ui_level: u8,
```

Comment:
i think this should be an enum w/ descriptive names for the variants -- otherwise, ui_level can be 256 different values and the user has no real understanding of what they all are

Reply:
I was going to solve this with a commentin the example config but I guess I could do an enum and a comment